torchvision/csrc/io/image/cpu/decode_jpeg.cpp

```diff
@@ -139,12 +200,28 @@ torch::Tensor decode_jpeg(const torch::Tensor& data, ImageReadMode mode) {
   auto tensor =
       torch::empty({int64_t(height), int64_t(width), channels}, torch::kU8);
   auto ptr = tensor.data_ptr<uint8_t>();
+  torch::Tensor temp_tensor;
+  if (cmyk_to_rgb_or_gray) {
+    temp_tensor = torch::empty({int64_t(width), 4}, torch::kU8);
```

Comment:
I think the intent could be a bit clearer if these were named `cmyk_line_tensor` instead of `temp_tensor` (same for the buffer).